lte/gateway/c/oai/tasks/mme_app/mme_app_bearer.c

```diff
@@ -1890,49 +1892,7 @@ int mme_app_handle_initial_context_setup_rsp_timer_expiry(
         mme_ue_s1ap_id);
     OAILOG_FUNC_RETURN(LOG_MME_APP, RETURNerror);
   }
-  ue_context_p->initial_context_setup_rsp_timer.id = MME_APP_TIMER_INACTIVE_ID;
-  ue_context_p->time_ics_rsp_timer_started         = 0;
-  /* *********Abort the ongoing procedure*********
-   * Check if UE is registered already that implies service request procedure is
-   * active. If so then release the S1AP context and move the UE back to idle
-   * mode. Otherwise if UE is not yet registered that implies attach procedure
-   * is active. If so,then abort the attach procedure and release the UE
-   * context.
-   */
-  ue_context_p->ue_context_rel_cause = S1AP_INITIAL_CONTEXT_SETUP_FAILED;
-  mme_app_desc_t* mme_app_desc_p     = get_mme_nas_state(false);
-  // Remove enb_s1ap_id_key from the hashtable
-  hashtable_uint64_ts_remove(
-      mme_app_desc_p->mme_ue_contexts.enb_ue_s1ap_id_ue_context_htbl,
-      (const hash_key_t) ue_context_p->enb_s1ap_id_key);
-
-  if (ue_context_p->mm_state == UE_UNREGISTERED) {
-    nas_emm_attach_proc_t* attach_proc =
-        get_nas_specific_procedure_attach(&ue_context_p->emm_context);
-    // Stop T3450 timer if its still runinng
-    if (attach_proc) {
-      nas_stop_T3450(attach_proc->ue_id, &attach_proc->T3450, NULL);
-    }
```

Comment:
can you elaborate on why this was done before, but no longer needed?

Reply:
I discussed with Pruthvi and got to know that, these changes are required because while implicit detach is in progress, there is possibility of 3450 timer expiry, which eventually retransmits Attach Accept message. 
So I will keep these changes